WordPress/WordPressTest/BlockEditorSettingsServiceTests.swift

```diff
@@ -22,9 +22,8 @@ class BlockEditorSettingsServiceTests: XCTestCase {
         mockRemoteApi = MockWordPressComRestApi()
         blog = BlogBuilder(context)
             .with(wordPressVersion: "5.8")
-            .with(isHostedAtWPCom: true)
+            .withAnAccount()
```

Comment:
@antonis @chipsnyder @emilylaguna I pinged you for a review to verify I resolved the various conflicts on this test properly. Thanks! 🙇‍♂️

Reply:
@mokagio Yup this looks good to me. Either version would capture the spirit of this test so we're good 👍